### PR TITLE
Reorganized and added macOS

### PR DESCRIPTION
**Pull request by CTKRocks**
Just added links to separate `.md` files for Debian and macOS.

### CODE REVIEW

**Comment by CTKRocks** [comment]
Sorry about that. Fixed the conflict.

File: README.md [hunk 1]
```diff
@@ -23,41 +23,16 @@ these steps:
...
-    sudo apt-get install libsdl1.2debian libfdt1 libpixman-1-0
+[Build on Debian Stretch](debian_build.md)
 
+[Build on macOS](mac_build.md)
 
 If you wish to build firmware to run on your device, you may also wish to
 consider a script like [buildfw.sh](buildfw.sh).  Running RebbleOS on hardware is
 currently out of scope for this document.
 
+> You need the [`snowy_fpga.bin`](http://emarhavil.com/~joshua/snowy_fpga.bin) and [`chalk_fpga.bin`](http://emarhavil.com/~joshua/chalk_fpga.bin) files to compile on their respective firmwares.
```
Comment (jwise) [inline README.md]: maybe 'ask in discord' for these?  would rather these links not be in the repo

File: README.md [hunk 2]
```diff
@@ -23,41 +23,16 @@ these steps:
...
-    cd ~/pebble-dev/pebble-sdk-4.5-linux64
-    virtualenv --no-site-packages .env
-    source .env/bin/activate
-    pip install -r requirements.txt
-    deactivate
-
-The Pebble emulator requires the following libraries.
-    
-    sudo apt-get install libsdl1.2debian libfdt1 libpixman-1-0
+[Build on Debian Stretch](debian_build.md)
```
Comment (jwise) [inline README.md]: (can we use a full sentence for index-documentation?)

File: debian_build.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
+# Building on Debian Stretch
```
Comment (jwise) [inline debian_build.md]: splitting this out seems like a good idea; move to doc/build-debian.md?

File: mac_build.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
+# Building on macOS
+
+The following builds RebbleOS on macOS:
```
Comment (jwise) [inline mac_build.md]: have you tested this from a clean OS X install?  this works for me but I'm not sure if I did anything special to my machine

**Comment by jwise** [comment]
can you rebase this on top of master to avoid the merge conflicts?

**Comment by CTKRocks** [comment]
Ok should be good now.

**Pull request merged**